MYabrv.bib

```diff
@@ -31,6 +31,7 @@ @String{ECMFA
 @String{ECOOP = "Proc.\ Europ.\ Conf.\ on Object-Oriented Programming (ECOOP)"}
 @String{ESEC = "Proc.\ Europ.\ Software Engineering Conf.\ (ESEC)"}
 @String{ESECFSE = "Proc.\ Europ.\ Software Engineering Conf./Foundations of Software Engineering (ESEC/FSE)"}
+@String{ESEM16 = "{Proceedings of the 10th ACM/IEEE International Symposium on Empirical Software Engineering and Measurement"}
```

Comment:
- For Booktitles, we use generalized (year-independent) abbreviations usually. Only Vamos is an exception because it changed from workshop to conference.
- We also abbreviate `Proceedings` and `International`.
- The abbreviation is missing in the end.
- This is a symposium and not a conference so please move to the resepective list (somewhere below in the file)

So please change to
`ESEM = Proc.\ of the Int'l Conf.\ Symposium on Empirical Software Engineering and Measurement (ESEM)`